### PR TITLE
#162412562 user signup

### PR DESCRIPTION
**Pull request by bl4ck4ndbr0wn**
#### What does this PR do?
Have Post endpoint to allow users to register
#### Description of Task to be completed?
Test POST /api/v2/auth/register and make them fail
Create register endpoints 
Test API endpoint to confirm that the register is working.
#### How should this be manually tested?
Clone the repo as bellow:
 ```
 git clone - b ft-user-signup-162412562 git@github.com: bl4ck4ndbr0wn/iReporter.git
```
Create a virtual environment and activate it
 ```
 python3 - m venv env
 source .env
 ```
 Install all the dependencies needed by
 ```
 pip install - r requirements.txt
 ```
 Run the flask application
 ```
  flask run
 ```
On postname try sending a request
```
* POST  /api/v2/auth/register a user
* :params
      {  
         “firstname” : String,
         “lastname” : String,
         “othernames” : String,
         “email” : String,
         ”phoneNumber” : String,
         “username” : String,
         “password” : String,
       }
```
#### Any background context you want to provide?
Create a postgresql database and name it ireporter.
#### What are the relevant pivotal tracker stories?
 [  # 162412562](https://www.pivotaltracker.com/n/projects/2226962)

### CODE REVIEW

File: app/tests/v2/integration/models/test_records.py [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
-    #     self.assertIsNone(record.find_by_id(1),
-    #                       "Found an Incident with id '1' before save_to_db")
-    #
-    #     record.save_to_db()
-    #
-    #     self.assertEqual(record.record_type, 'red-flag')
-    #     self.assertIsNotNone(Incident.find_by_id(1),
-    #                          "Did not find an Incident with"
-    #                          " id '1' after save_to_db")
+    def test_crude(self):
```
Comment (thoth-ky) [inline app/tests/v2/integration/models/test_records.py]: what is crude?

Comment (bl4ck4ndbr0wn) [reply app/tests/v2/integration/models/test_records.py]: create, update, delete, read

Comment (thoth-ky) [reply app/tests/v2/integration/models/test_records.py]: crud, not crude. :)

Comment (bl4ck4ndbr0wn) [reply app/tests/v2/integration/models/test_records.py]: ok.

File: app/__init__.py [hunk 2]
```diff
@@ -3,12 +3,6 @@
 from flask_jwt_extended import JWTManager
 from instance.config import config
 
-from app.api.v1.views.incident import (RedFlagRecords,
-                                       RedFlagRecord,
-                                       RedFlagRecordLocation,
-                                       RedFlagRecordComment)
-from app.api.v1.views.auth import (SignIn, SignUp)
-
 jwt = JWTManager()
```
Comment (thoth-ky) [inline app/__init__.py]: do away with jwt-extended

**Review by thoth-ky — APPROVED** [review]
Looks good

**Review by thoth-ky — APPROVED** [review]
Good work.

**Pull request merged**